lighthouse-core/scripts/lantern/run-once.js

```diff
@@ -8,9 +8,9 @@
 
 const fs = require('fs');
 const path = require('path');
-const PredictivePerf = require('../../../lighthouse-core/audits/predictive-perf.js');
-const Simulator = require('../../../lighthouse-core/lib/dependency-graph/simulator/simulator.js');
-const traceSaver = require('../../../lighthouse-core/lib/lantern-trace-saver.js');
+const PredictivePerf = require('../../audits/predictive-perf.js');
+const Simulator = require('../../lib/dependency-graph/simulator/simulator.js');
+const traceSaver = require('../../lib/lantern-trace-saver.js');
```

Comment:
the only path changes that aren't adding a suffix because these files are already under `lighthouse-core/` :)

The lint rule actually won't flag or fix this case normally (since the paths do have a file extension and they resolve to a file), but these were changed when I was testing on the reverted #9006 (which added `.js` to them) and it seems worth keeping the simplified form